docker/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  snp_blending:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - TAG=humble-master
+    environment:
+      DISPLAY: $DISPLAY
+      XAUTHORITY: $XAUTHORITY
+      NVIDIA_DRIVER_CAPABILITIES: all
+      ROS_LOG_DIR: /tmp
+    container_name: snp_blending
+    image: ghcr.io/ros-industrial-consortium/snp_blending:humble-master
+    stdin_open: true
+    tty: true
+    network_mode: host
+    privileged: true
+    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - /etc/hosts:/etc/hosts
+      - /etc/passwd:/etc/passwd:ro
+      - /etc/group:/etc/group:ro
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
```

Add micro-ROS agent for MotoROS2 driver

```suggestion
              capabilities: [gpu]
  # Driver
  micro-ros-agent:
    environment:
      ROS_LOG_DIR: /tmp
      ROS_LOCAL_HOST_ONLY: 1
    container_name: micro-ros-agent
    image: microros/micro-ros-agent:humble
    command: udp4 --port 8888
    stdin_open: true
    tty: true
    privileged: true
    network_mode: host
```